docs/src/using-turing/dynamichmc.md

```diff
@@ -7,7 +7,7 @@ title: Using DynamicHMC
 Turing supports the use of [DynamicHMC](https://github.com/tpapp/DynamicHMC.jl) as a sampler through the `DynamicNUTS` function.
 
 
-`DynamicNUTS` is not appropriate for use in compositional inference. If you intend to use [Gibbs](http://turing.ml/docs/library/#-turinggibbs--type) sampling, you must use Turing's native `NUTS` function.
+`DynamicNUTS` is not appropriate for use in compositional inference. If you intend to use [Gibbs](https://turing.ml/dev/docs/library/#Turing.Inference.Gibbs) sampling, you must use Turing's native `NUTS` function.
```

Comment:
Maybe a relative link here? 

```suggestion
`DynamicNUTS` is not appropriate for use in compositional inference. If you intend to use [Gibbs](docs/library/#Turing.Inference.Gibbs) sampling, you must use Turing's native `NUTS` function.
```

Reply:
I changed your suggestion to handle the different site URLs.

Reply:
thanks, the [link](https://turing.ml/docs/library/#Turing.Inference.Gibbs) seems broken without `dev`, due to #1157.

Reply:
maybe add the `dev` for now, until #1157 is fixed. apologies for the noise.

Reply:
Actually, is there a way to add version number automatically after `{{site.baseurl}}`?

Reply:
`site.baseurl` handles all the versioning, including `dev`.

Reply:
many thanks!